ReqIFSharp/AttributeValue/AttributeValueDate.cs

```diff
@@ -132,7 +132,14 @@ protected override void SetAttributeDefinition(AttributeDefinition attributeDefi
         public override void ReadXml(XmlReader reader)
         {
             var value = reader["THE-VALUE"];
-            this.TheValue = XmlConvert.ToDateTime(value, XmlDateTimeSerializationMode.Utc);
+            try
+            {
+                this.TheValue = XmlConvert.ToDateTime(value, XmlDateTimeSerializationMode.Utc);
+            }
+            catch (Exception ex)
+            {
+                System.Diagnostics.Debug.WriteLine(ex.Message);
```

Comment:
can you explain why you use `System.Diagnostics.Debug.WriteLine`?

Reply:
Empty catch block it is bad practice, I don't find any logs in program so I write it to debug console